src/tensorflow_translators.hpp

```diff
@@ -6,11 +6,8 @@
 
 #include <openvino/frontend/node_context.hpp>
 #ifdef OpenVINO_Frontend_TensorFlow_FOUND
-#include <openvino/frontend/tensorflow/node_context.hpp>
```

Comment:
do we need conditional compilation depending on TF FE availability?

Reply:
Yes, le us have it. May be someone does not want to include it

Reply:
but you have removed definition from cmake => this code is always OFF

Reply:
I removed only linking TF FE. OpenVINO_Frontend_TensorFlow_FOUND is coming from OV cmakes as I understand.

Reply:
but target_compile_definition is removed

Reply:
I have just restored it. Thanks for clarification.

Reply:
IMO, we need to remove conditions here, because overhead is quite small - 1-2 files need to be recompiled.
But in this case, we will have unified binary

Reply:
Removed